packages/types/src/documents/global/config.ts

```diff
@@ -111,18 +111,26 @@ export interface SCIMInnerConfig {
 
 export interface SCIMConfig extends Config<SCIMInnerConfig> {}
 
-export type AIProvider = "OpenAI" | "Anthropic" | "TogetherAI" | "Custom"
+export type AIProvider =
+  | "OpenAI"
+  | "Anthropic"
+  | "AzureOpenAI"
+  | "TogetherAI"
+  | "Custom"
+
+export interface ProviderConfig {
+  provider: AIProvider
+  isDefault: boolean
+  isBudibaseAI?: boolean
```

Comment:
This field is new, and much more explicitly labels a config as Budibase AI. We were previously inferring this from other fields. Budibase AI config is always enriched at runtime, so this won't be stored anywhere.